common/lib/xmodule/xmodule/js/src/video/03_video_player.js

```diff
@@ -549,6 +549,9 @@ function(HTML5Video, HTML5HLSVideo, Resizer, HLS, _, Time) {
     function onUnstarted() { }
 
     function handlePlaybackQualityChange(value) {
+        console.log(
+            `[Video info]: changing playback quality from ${this.videoPlayer.player.getPlaybackQuality()} to ${value}`
+        );
```

Comment:
Is this method called for both HLS and YT videos? Can you add some sample logs of this in the PR description against an HLS video?